en/test_and_ci/continous_integration.md

```diff
@@ -15,7 +15,7 @@ Travis-ci also has a macOS px4_sitl build which includes testing.
 
 ## Semaphore
 
-[Semaphore](https://semaphoreci.com/px4/PX4-Autopilot) is primarily used to compile changes for the Qualcomm Snapdragon platform, but also serves as a backup to Travis-ci using the same [px4io/px4-dev-base](https://hub.docker.com/r/px4io/px4-dev-base/) docker image.
+[Semaphore](https://semaphoreci.com/px4/firmware) is primarily used to compile changes for the Qualcomm Snapdragon platform, but also serves as a backup to Travis-ci using the same [px4io/px4-dev-base](https://hub.docker.com/r/px4io/px4-dev-base/) docker image.
```

Comment:
@bkueng This appears to not have been updated to PX4-Autopilot and last update was 2017. OK to just delete this whole section?

Are any of the other sections known to not be used and can be deleted too (we know this doc is largely pointless)?

Reply:
None of these 3 are used anymore, and you can remove them all. Indeed completely outdated. We use github actions and a Jenkins server.